app/spec/controllers/caseworker/entries_controller_spec.rb

```diff
@@ -27,5 +24,17 @@
         expect(unescaped_body).to include("Log in with your LAN ID")
       end
     end
+
+    context "when state is disabled" do
+      before do
+        stub_site_config_value("sandbox", "staff_portal_enabled", false)
+      end
+
+      it "should show redirect to the root page" do
```

Comment:
nit: https://www.betterspecs.org/#should

e.g.
```suggestion
      it "redirects to the root page" do
```